### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by CynthiKite**
docs: Updated reference to terminology to IndyKite glossary
docs: Updated documentation link
docs: Removed indykite.one references

# Checklist

- [ ] `make test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added

### CODE REVIEW

File: README.md
Comment (cowan-macady) [inline README.md:1]: - If you want to add requirements, you need a customer and a service account
- protobuf is not necessary: the generated files are integrated in the SDK

**Review by cowan-macady — CHANGES_REQUESTED** [review]

File: README.md
Comment (CynthiKite) [reply README.md:1]: I removed protobuf. 

I'm not sure what you mean by add requirements.

Comment (cowan-macady) [reply README.md:1]: line 20 (inside the requirements section)

Comment (cowan-macady) [reply README.md:1]: you forgot the customer

**Pull request closed without merging**